justfile

```diff
@@ -11,7 +11,10 @@ prefetch:
 test:
     cargo fmt --all -- --check
     cargo clippy --workspace
-    cargo test --workspace
+    cargo test --workspace --lib
```

Comment:
I believe in the README we mention that using `cargo` instead of `just` is also fine. Perhaps we should note that `cargo test` will run integration tests so it's advisable to run with the `--lib` option when testing locally (unless of course the dev wants to run integration tests too)